src/main/webapp/timestamps.js

```diff
@@ -12,35 +12,34 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+const MILLISECONDS_PER_SECOND = 1000;
 const MINUTES_TO_SECONDS = 60;
 const HOURS_TO_SECONDS = 360;
 
 /**
- * Coverts a Date representing a video {@code timestamp} into a string.
+ * Converts a video {@code timestampMs} in milliseconds into a string.
  */
-function timestampToString(timestamp) {
-  const date = new Date(timestamp);
-  const seconds = date.getSeconds().toString().padStart(
+function timestampToString(timestampMs) {
+  const date = new Date();
+  date.setTime(timestampMs);
+  const seconds = date.getUTCSeconds().toString().padStart(
       /* targetLength= */ 2, /* padString= */ '0');
-  const minutes = date.getMinutes().toString().padStart(
+  const minutes = date.getUTCMinutes().toString().padStart(
       /* targetLength= */ 2, /* padString= */ '0');
-  if (date.getHours() == 0) {
+  if (date.getUTCHours() == 0) {
     return `${minutes}:${seconds}`;
   }
   // We don't pad hours because lectures won't need two digits for hours.
-  const hours = date.getHours().toString();
+  const hours = date.getUTCHours().toString();
   return `${hours}:${minutes}:${seconds}`;
 }
 
 /**
- * Returns the total number of seconds since the start of a video at a
- * certain {@code timestamp}.
+ * Converts a number of `seconds` to an integer number of milliseconds.
```

Comment:
I think this comment an be made simpler by saying: Converts seconds to milliseconds.

Reply:
Changed